src/app/lib/mainsail/ledger.service.ts

```diff
@@ -114,6 +114,16 @@ export class LedgerService {
 		return [`0x`, r, s, v.toString(16)].join("");
 	}
 
+	public async isEthBasedApp() {
+		try {
+			const path = `m/44'/60'/0'/0/0`;
```

Comment:
can we add a comment why this check works for our usecase? to avoid forgetting about it in the future. Each ledger app defines the paths that are allowed, and the old ark app only accepts 111 and 1 for mainnet and devnet so using 60 errors